manifests/w/W1HKJ/Flrig/2.0.02/W1HKJ.Flrig.locale.en-US.yaml

```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: W1HKJ.Flrig
+PackageVersion: 2.0.02
+PackageLocale: en-US
+Publisher: W1HKJ Software

```

Comment:
I agree. It looks like it is correct [in Fldigi](https://github.com/microsoft/winget-pkgs/blob/master/manifests/w/W1HKJ/Fldigi/4.1.27/W1HKJ.Fldigi.locale.en-US.yaml#L14). However, a small nitpick: It should be _Flrig_, not _FLRIG_.

Reply:
@AsciiWolf The installer put this value with this casing in the Registry. The value should match whatever is written to Registry's Publisher value that you can see in `appwiz.cpl` / Control Panel after installation of the package. The value must match for good upgrade matching/correlation for the WinGet client.

Reply:
Oops, I take it back. It's only the Copyright, not the actual Publisher in Fldigi. Anyway, I believe that the publisher actually should be "W1HKJ Software" (since it is the original author and the one who publishes this software) and copyright should be "Flrig developers" (since other Open Source community members are also helping develop this software).

Reply:
@mdanish-kh Then it should be fixed in Fldigi as well, not only in Flrig.

Reply:
> @mdanish-kh Then it should be fixed in Fldigi as well, not only in Flrig.

If the value written in the manifest does not match Control Panel / Registry, then yes it should

Reply:
@CoolPlayLin Any chance you could change the Publisher for [Fldigi](https://github.com/microsoft/winget-pkgs/tree/master/manifests/w/W1HKJ/Fldigi/4.1.27) to `Fldigi developers` and for Flrig to `Flrig developers` + maybe also change the Copyright for Fldigi to `Copyright (C) 2023 W1HKJ Software`? Thanks!